errors.yaml

```diff
@@ -0,0 +1,50 @@
+---
+errors:
+  - code: index_creation_failed
+    description: "An error occurred while trying to create an index, check out our guide on [index creation](https://docs.meilisearch.com/guides/main_concepts/indexes.html)"
+  - code: index_already_exists
+    description: "An index with this UID already exists, check out our guide on [index creation](https://docs.meilisearch.com/guides/main_concepts/indexes.html)"
+  - code: invalid_index_uid
+    description: "There is an error in the provided index format, check out our guide on [index creation](https://docs.meilisearch.com/guides/main_concepts/indexes.html)"
+  - code: index_not_accessible
+    description: "An internal error occurred while trying to access the requested index."
+  - code: invalid_state
+    description: "An error that occurs when the database is in an invalid state. Deleting the database and re indexing should solve the problem."
+  - code: missing_primary_key
+    description: "All documents must be identified with a unique key. MeiliSearch will try to infer the document field to use it as a primary key. If it can't infer it, the primary key must be specified manually, and this error is returned. . See  our [guide on primary keys](https://docs.meilisearch.com/guides/main_concepts/documents.html#structure)."
```

Comment:
```suggestion
    description: "All documents must be identified with a unique key. MeiliSearch will try to infer the document field to use it as a primary key. If it can't infer it, the primary key must be specified manually, and this error is returned. See  our [guide on primary keys](https://docs.meilisearch.com/guides/main_concepts/documents.html#structure)."
```